src/Microsoft.VisualBasic/tests/VB/TextFieldParserTests.vb

```diff
@@ -0,0 +1,180 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System.Runtime.CompilerServices
+Imports Microsoft.VisualBasic.FileIO
+Imports Xunit
+
+Namespace Microsoft.VisualBasic.Tests.VB
+    Public NotInheritable Class TextFieldParserTests
+        Inherits FileIOTestBase
+
+        Private Const HeaderLine As String = "seq,oneword,age,street,city,state,zip,dollar,pick,date,paragraph"
+
+        Private Const CSVData As String =
+                    HeaderLine & vbCrLf &
+            "1,Test,99,Two Words,Rumnoman,KS,80586,$8333.85,GREEN,09/28/1929,""Ipmadaw udoteac vu ote ozehop ane lujat, lar buh cugi lef owici tat liz cogde.
+Fatigiug kojitfu taapi iz alujok me zimipa koz latkekij vem fo si cepzizhub li por ejcirlu.
+Ewujajlu mivec tooju ad bu cowic irtafit ehaoca vojhehfo aztidun zo wezecmo abe muz wikhutwen idce.
+Vesu sejawga tef lahi dirueg si uwmac bidiw nowidza daime sapmim ki casdun urokir tawdac rahaw beiweed."""
+
+        Private Shared ReadOnly FixedFieldWidths() As Integer = {5, 10, 6, -1}
+        Private Shared ReadOnly FixedFormatData() As String = {
+            "Err  1001                  Cannot access resource.",
+            "Err  2014                  Resource not found.",
+            "Acc  10/03/2009 User1      Administrator.",
+            "Err  0323                  Warning: Invalid access attempt.",
+            "Acc  10/03/2009 User2      Standard user.",
+            "Acc  10/04/2009 User2      Standard user."
+        }
```

Comment:
Please move `FixedFieldWidths` and `FixedFormatData` to the one test where the values are used.